artifacts/scripts/construct.sh

```diff
@@ -96,6 +96,32 @@ git config user.name "$GIT_COMMITTER_NAME"
 echo "Running garbage collection."
 git config gc.pruneExpire 3.days.ago
 git gc --auto
+
+# Remove corrupted lines in .git/packed-refs
+# Lines not starting with '^' caret are printed as-is. Lines
+# that do not start with a caret are printed ONLY if the previous
+# line did not start with a caret
+echo "Cleaning .git/packed-refs"
+cp packed-refs packed-refs.bak
+awk '
+NR == 1 {
+    prev_line_start_with_caret = 0
+    print
+}
+/^[^^]/ {
+    prev_line_start_with_caret = 0
+    print
+}
+/^\^/ {
+    if (!prev_line_start_with_caret) {
+        print
+        prev_line_start_with_caret = 1
+    } else {
+        prev_line_start_with_caret = 1
+    }
+}
+' packed-refs.bak > packed-refs
```

Comment:
The `packed-refs.bak` needs to be removed from the repo after performing the cleaning, else it may change the output of `git status`

Reply:
doing this!

Reply:
I couldnt find where the `.git/packed-refs` was being removed. Since its not in the `.git` folder it wont affect the repo state, but prefer not having temporary files kept.